### ISSUE TITLE
Return pcb (vmID) param

### ISSUE THREAD

**Issue by Vadimda** [FreeRDP/FreeRDP-WebConnect#125]
Please, return pcb (vmID) param to wsgate

**Comment by c64cosmin** [comment]
Can you explain more what your issue is?

**Comment by Vadimda** [comment]
Free RDP WebConnect from Cloudbase.it (
https://cloudbase.it/freerdp-html5-proxy-windows/) contains field PCB(vmID)
that allows connection to hyper-v virtual machines.
Can you add this functionality to Free RDP WebConnect?

2016-01-30 23:39 GMT+03:00 c64cosmin <notifications@github.com>:

**Comment by c64cosmin** [comment]
Are you using it with Openstack?

**Comment by Vadimda** [comment]
No. Just with Hyper-V.

2016-01-30 23:58 GMT+03:00 c64cosmin <notifications@github.com>:

**Comment by c64cosmin** [comment]
You just need to enter under advanced options in the login menu.

**Comment by Vadimda** [comment]
Yes. But there isn't PCB(vmID) field in advanced options. (FreeRDP
WebConnect Version 1.0.0.167)

2016-01-31 0:13 GMT+03:00 c64cosmin <notifications@github.com>:

**Comment by c64cosmin** [comment]
Can you tell me how you installed WebConnect, also could you send me a screen shot of the advanced menu?

**Comment by Vadimda** [comment]
Installed from FreeRDP-WebConnect-1.0.0.167-Setup.exe

Cloudbase FreeRDP WebConnect installed from FreeRDPWebConnect_Beta.msi

2016-01-31 0:21 GMT+03:00 c64cosmin <notifications@github.com>:

**Comment by c64cosmin** [comment]
![image](https://cloud.githubusercontent.com/assets/6196122/12698608/e836dc2e-c7a9-11e5-8950-106248ce508c.png)
This is how your advanced menu should look like.

**Comment by Vadimda** [comment]
Yes. It's in Free RDP Webconnect from cloudbase. But they don't continue
development.

But  Free RDP Webconnect from http://freerdp.net/ continue development, but
haven't PCB(vmID) field. I ask to add this field to advanced menu.

2016-01-31 0:34 GMT+03:00 c64cosmin <notifications@github.com>:

**Comment by c64cosmin** [comment]
My contribution is only for this project (WebConnect).
I suppose you need the "/pcb:" command line parameter for freerdp.
https://github.com/FreeRDP/FreeRDP/wiki/CommandLineInterface

**Comment by alexpilotti** [comment]
Hi @Vadimda, we are definitely continuing the development, as you can see in all the activity in this project, including @c64cosmin contributions. :-)

Is there something in particular that you are looking for?

**Comment by Vadimda** [comment]
I'm sorry. Can you close and delete this issue. I see that
https://github.com/FreeRDP/FreeRDP-WebConnect is newer that in
http://freerdp.net/ . I's my mistake.

Where can I download latest windows compiled FreeRDP-WebConnect? Or how can
I compile from sources in windows?

2016-01-31 0:43 GMT+03:00 c64cosmin <notifications@github.com>:

**Comment by alexpilotti** [comment]
@Vadimda, there's an automated build system that generates a new MSI anytime a patch lands in the repo: https://www.cloudbase.it/downloads/FreeRDPWebConnect_Beta.msi

Once a stable release it's ready, it becomes available on the web site.

**Comment by Vadimda** [comment]
Thank you very much!!!

2016-01-31 0:54 GMT+03:00 Alessandro Pilotti <notifications@github.com>:
